prow/plugins/updateconfig/updateconfig_test.go

```diff
@@ -1428,8 +1428,8 @@ func testUpdate(clients localgit.Clients, t *testing.T) {
 			t.Fatalf("Failed to clone: %v.", err)
 		}
 		defer func() {
-			if err := c.Clean(); err != nil {
```

Comment:
In the test its actually correct to clean everything

Reply:
Exactly. Fixed.